tests/common.go

```diff
@@ -6161,7 +6161,14 @@ func CollectLogsFromPods(testCaseName string, podLabel map[string]string, namesp
 
 	// Check to handle cloud based deployment with 0 master nodes
 	if len(node.GetMasterNodes()) == 0 {
-		log.Warnf("Skipping pod log collection for pods with [%s] label in test case [%s]", logLabel, testCaseName)
+		log.Warnf("Skipping pod log collection for pods with [%s] label in test case [%s] as it's cloud cluster", logLabel, testCaseName)
+		return
+	}
+
+	// In case of ocp if ssh_key is given skip log collection
+	sshKey := os.Getenv("TORPEDO_SSH_KEY")
```

Comment:
even for normal vsphere runs many PX jobs pass TORPEDO_SSH_KEY.

Reply:
What we observed was that our runs were failing on ocp due to it not being able to ssh to collect log pods. This blocks the upgrade jobs from running. Currently from the backup end we don't have a job which has this parameter other than the ocp job.

Reply:
generally we pass nodedriver=openshift for ocp jobs. can you use that variable to check if it is ocp or not

Reply:
I checked this job which you share the problem is I don't see `schedular=openshift` being used anywhere in the code
https://jenkins.pwx.dev.purestorage.com/job/3.1.0-dev-jobs/job/ocp-dev-2/build?delay=0sec

Reply:
Got a one which @lsrinivas-pure  is using the flag https://jenkins.pwx.dev.purestorage.com/job/Torpedo/view/vsphere-dmthin/job/OCP-PoolExpand-1/36/consoleFull. Changing the code.

Reply:
Done